Makefile

```diff
@@ -49,6 +49,9 @@ test_s3_pipeline: require_vars ## Deploy a pipeline that tests S3 access to conc
 deploy_pipeline: require_vars ## Deploy the CF deployment pipeline
 	@ci/deploy_pipeline.sh
 
+pipeline_of_destruction: require_vars ## Deploy the CF destruction pipeline
```

Comment:
If it's going to be a makefile target then it needs a better name - something like `deploy_destruction_pipeline`